docs/source/markdown/options/cidfile.read.md

```diff
@@ -0,0 +1,4 @@
+#### **--cidfile**=*file*
+
+Read container ID from the specified *file* and <<subcommand>> the container.
```

Comment:
Here `<<subcommand>>` is perfect except for `podman rm`, where the sentence becomes `and rm the container` (instead of "remove"). IMO the dup-removal makes this worthwhile. Same with `--pod-id-file` below.